recipes/sumaclust/meta.yaml

```diff
@@ -0,0 +1,24 @@
+{% set hash_type = "sha256" %}
+{% set hash_value = "a7c122ff90671d8589deb05a1a8b008e8f439103bb0b7eca48c3d6da205cf2f2" %}
+
+package:
+  name: sumaclust
+  version: "1.0.31"
+
+source:
+  url: https://git.metabarcoding.org/obitools/sumaclust/uploads/59ff189079b9e318f07b9ff9d5fee54b/sumaclust_v1.0.31.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+
+test:
+  commands:
+    - sumaclust -h > /dev/null
+
+about:
+  summary: Sumaclust. For more information see url.
```

Comment:
A short summery would be nice.

Reply:
They have the longest 'summary'. Didn't want to butcher it but will try to do it justice. :) Thanks.